root/Makefile

```diff
@@ -144,6 +144,8 @@ gogenerate:: pre-gogenerate
 ## gobuild:         build application binary
 .PHONY: gobuild
 gobuild:
+	# ensure that we have secrets for the build phase
+	if [[ -z $$SKIP_DEVCONFIG ]]; then ./scripts/shell-wrapper.sh devconfig.sh; fi
```

Comment:
Ehhhh, the reason we made this a separate step (`devserver`) was because `devconfig.sh` is slow to run. I don't like making user's endure that on every local build

Reply:
It doesn't seem that slow. I don't know if there is any other way to ensure this 100%.

What do you think about making this a little easier by providing `make devconfig`? At least that way it's a little more intuitive what to run.

Reply:
`make devconfig` is good for now, hopefully this'll all be moot after devspace anyways 🙏

Reply:
Unfortunately, devspace still relies on the same mechanism.